app/templates/gulpfile.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+// Generated on <%= (new Date).toISOString().split('T')[0] %> using <%= pkg.name %> <%= pkg.version %>
+
+var gulp = require('gulp'),
+    $ = require('gulp-load-plugins')(),
+    browserSync = require('browser-sync'),
+    runSequence = require('run-sequence'),
+    del = require('del'),
+    neat = require('node-neat').includePaths;
+
+var production = false,
+    paths = {
+      app:     'app',
+      html:    'app/**/*.html',
+      styles:  'app/styles/**/*.scss',<% if (jsPre === 'coffeescript') { %>
+      scripts: 'app/scripts/**/*.coffee',<% } else { %>
+      scripts: 'app/scripts/**/*.js',<% } %>
+      images:  'app/images/**/*.{png,gif,jpg,jpeg,svg}',
+      fonts:   'app/fonts/**/*.{eot*,otf,svg,ttf,woff}',
+      vendor:  'vendor'
+    };
+
+gulp.task('html', function (cb) {
+  var config = (production) ? '_config.yml,_config.build.yml' : '_config.yml',
+      dest   = (production) ? 'dist' : '.tmp';
+
+  var spawn  = require('child_process').spawn,
+      jekyll = spawn('jekyll', ['build', '-q', '--config', config, '-s', paths.app, '-d', dest], { stdio: 'inherit' });
+
```

Comment:
See comment from line nine of this file. This style of variable assignment is different than the one in index.js.